### PR TITLE
translate database-mariadb-server.md to spanish

### PR DESCRIPTION
**Pull request by execion**
## Author checklist (to be completed by original Author)
- [x] Is this document a good fit for the Rocky project ?
- [x] Is this a non-English contribution? 
- [ ] Title and Author MetaTags have been inserted into the document 
- [x] If applicable, steps and instructions have been tested to work on a real system
- [x] Did you perform an initial self-review to fix basic typos and grammatical correctness

## Rocky Documentation checklist  (to be completed by Rocky team) 
- [x] 1st Pass (Check that document is good fit for project and author checklist completed)
- [ ] 2nd Pass (Technical Review - check for technical correctness) 
- [ ] 3rd Pass (Basic Editorial Review)
- [ ] 4th Pass (Detailed Editorial Review and Peer Review)
- [ ] 5th Pass (Include document in TOC)
- [x] Final pass/approval (Final Review)

### CODE REVIEW

**Comment by calderds** [comment]
Thanks for you r contribution! If you can get another spanish speaker to edit your document for grammar, I'd be happy to include this in the main repo!

File: es/rocky/8/manuales/database_mariadb-server.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
+# Base de datos mariadb-server
+
+## Pre-requisitos
+
+* Un servidor con Rocky Linux.
+* Estaremos usando _"vi"_ como editor de linea de comandos, por lo que es necesario saber usarlo
+* Una gran facilidad para el uso de líneas de comando en la terminal, revisar los logs (registros de errores), y otros aspectos generales de un administrador de sistema
+* Será útil algún conocimiento sobre _mariadb-server_
+* Toma en consideración que todos los comandos deben ser ejecutados super usuario o con _sudo_
```
Comment (kgurnani) [inline es/rocky/8/manuales/database_mariadb-server.md:9]: ejecutados **con** super usuario

File: es/rocky/8/manuales/database_mariadb-server.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+Para fortalecer la seguridad en _mariadb-server_ necesitamos primero habilitar el servicio:
+
+> `systemctl enable mariadb`
+
+Y luego inicializarlo:
+
+> `systemctl start mariadb`
+
+Para finalmente ingresar esté comando:
```
Comment (kgurnani) [inline es/rocky/8/manuales/database_mariadb-server.md:35]: este no lleva tilde

**Comment by execion** [comment]
I've finished with the corrections indicated by @kgurnani.

**Pull request merged**